src/govuk/all.test.js

```diff
@@ -40,10 +40,11 @@ describe('GOV.UK Frontend', () => {
 
       const GOVUKFrontendGlobal = await page.evaluate(() => window.GOVUKFrontend)
 
-      var components = Object.keys(GOVUKFrontendGlobal).filter(method => method !== 'initAll')
+      const exports = Object.keys(GOVUKFrontendGlobal)
```

Comment:
This change makes sense to me but the test description ('exports components') isn't quite right now we're also checking for `initAll` and `i18n`.

Reply:
That's absolutely fine

I've put back the `.filter()` and added a separate test for `it('exports 'I18n' function')`

I've made sure it's in keeping with `it('exports 'initAll' function')`

Is that alright?

Reply:
Neat 👍🏻